Dockerfile

```diff
@@ -11,7 +11,8 @@
 
 # To build with a different base image
 # please run `docker build` using the `--build-arg PYTORCH_IMAGE=...` flag.
-ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:24.03-py3
+# ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:24.03-py3
+ARG PYTORCH_IMAGE=gitlab-master.nvidia.com:5005/dl/dgx/pytorch:24.08-py3-stage
```

Comment:
It would be better to pass this arg in some selected CI run, instead of overriding it with an internal image that is only accessible inside NVIDIA.

Reply:
Right, I was expecting to change it to nvcr.io 24.08 once it's out ... I suppose we can live with 24.07 for now, vista3d test in the zoo may fail though.

Reply:
Hi @borisfom, since updating the base image will need more tests, let's wait until 24.08. Once it's available, I'll update the base image and trigger additional CI tests. It's just a few days away from the release.

Reply:
@KumoLiu : 24.08 is out.

Reply:
Cool, will test it.